contents/blog/the-posthog-array-1-41-0.md

```diff
@@ -0,0 +1,165 @@
+
+![view recordings from anywhere1](../images/blog/array/1-41-0-view-recordings-anywhere1.png)
+
+![view recordings from anywhere2](../images/blog/array/1-41-0-view-recordings-anywhere2.png)
+
+You can now view session recordings from different places within PostHog.
+
+Curious about how a specific person is interacting with your app? Navigate to a person detail page and check out their recordings. Want to see recordings for a specific event or action you've created? Check out the new view recordings button you can find in the event's detail page or from the events table.
+
+### New: Change your email yourself
+
+![change your password](../images/blog/array/1-41-0-password.png)
+
+Finally, we can release one of our most requested features: the ability to change the email address attached to your account, without contacting support. All you have to do is select your profile picture in the top right and access your account settings. 
+
+### New: Hedgehog mode
+
+![hedgehog mode](../images/blog/array/1-41-0-hedgehog-mode.png)
+
+For a while now, we've been having a hard time explaining to our families what we do for a living. This makes it even harder.
+
+### Improved: Recordings interface
+
+![recordings interface](../images/blog/array/1-41-0-recordings-interface.png)
+
+We've heard feedback recently that session recording was incredibly useful, but didn't spark much joy for those who used it. So, we've overhauled the entire interface for session recordings to make it easier to use and to help you find relevant recordings faster. 
+
+We think session recording feels like an entirely new experience now, so check it out. Now's the perfect time to explore [the console log too](/manual/recordings#console-logs-recording-beta)!
+
+### One more thing: Site apps 
+
+![site-apps](../images/blog/array/1-41-0-site-apps.gif)
+
+We're testing a new big (beta) thing: site apps. You need to **manually opt in** to enable this feature by configuring your `posthog-js` initialization to include `opt_in_site_apps: true`. Once you do, PostHog will be able to inject code onto your website through `posthog-js`. We've put together [a tutorial that explains how to make a site app](/tutorials/build-site-app) if you're interested. 
+
+Site apps can be useful for a number of potential tasks, such as displaying feedback forms, posting service update banners, or making it rain pineapples. No, we're not really sure why we made the last one either. 
+
+### Other improvements & fixes
+
+You think that's it? Not by a long shot! Version 1.41 also adds hundreds of other improvements and fixes, including...
+
+- **Improvement:**  You can now [send analytics events from GitHub actions, to PostHog](https://github.com/PostHog/posthog-github-action)
+- **Improvement:**  We have revamped our timezone system! We've squashed various bugs and improved the interval grouping to be more in line with expectations when filtering on dates.
+- **Improvement:** 
```

Comment:
Not sure if something else should have been here, but I think we should mention https://github.com/PostHog/posthog/pull/12302:
```suggestion
- **Fix:** WAU/MAU aggregation in Trends was quietly always grouped by day, even if a different interval (hour/week/month) was selected. Additionally, those modes showed zero users for periods with no relevant events, even if the real count should have been non-zero due to WAU/MAU being a trailing count. Both issues are now fixed, and we've expanded our test coverage of those aggregation modes to ensure their results are accurate going forward.